tests/runtime/test_runtime_metrics_api.py

```diff
@@ -198,7 +198,7 @@ def test_runtime_metrics_enable_environ(monkeypatch, environ):
         RuntimeMetrics.disable()
 
 
-@pytest.mark.subprocess(parametrize={"DD_TRACE_EXPERIMENTAL_RUNTIME_ID_ENABLED": ["true", "false"]})
+@pytest.mark.subprocess(parametrize={"DD_TRACE_RUNTIME_ID_ENABLED": ["true", "false"]})
```

Comment:
Should we parametrize on the env name as well? so we can assert that `DD_TRACE_EXPERIMENTAL_RUNTIME_ID_ENABLED` continues to work?

Reply:
I didn't prioritize this given it was experimental. But I'll do it.